File: Getting-Practice-with-Python-and-Javascript/4- Functions/Javascript/Excercises/program-6.js
// 6.  Crea una función que tome una lista de números y devuelva el promedio de los números.


let lista_numeros = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]

const average = (lista) => {
    const sum = lista.reduce((total, currentValue) => total + currentValue, 0)
    let resultado = sum / lista.length
    console.log(parseInt(resultado))
}


average(lista_numeros)

File: Getting-Practice-with-Python-and-Javascript/4- Functions/Python/Excercises/program-6.py
# 6.  Crea una función que tome una lista de números y devuelva el promedio de los números.

lista_numeros = [1,2,3,4,5,6,7,8,9,10,11]

def average(lista):
    resultado = sum(lista) / len(lista)
    print(int(resultado))


average(lista_numeros)